ovs/lib/albacontroller.py

```diff
@@ -304,24 +320,30 @@ def remove_cluster(alba_backend_guid):
 
         cluster_removed = False
         for abm_service in albabackend.abm_services:
-            ip = abm_service.service.storagerouter.ip
-            service_name = abm_service.service.name
             if cluster_removed is False:
-                ArakoonInstaller.delete_cluster(service_name, ip)
-                cluster_removed = True
+                abm_metadata = ArakoonInstaller.get_arakoon_metadata_by_cluster_name(cluster_name=abm_service.service.name)
+                abm_metadata.unclaim()
```

Comment:
You should not unclaim and then delete the cluster, as the cluster might already be in use by another backend.